tests/ExceptionTest.php

```diff
@@ -2,13 +2,20 @@
 
 namespace atk4\dsql\tests;
 
+use atk4\dsql\Connection;
 use atk4\dsql\Expression;
+use atk4\dsql\Query;
 
 /**
  * @coversDefaultClass \atk4\dsql\Exception
  */
 class ExceptionTest extends \atk4\core\PHPUnit_AgileTestCase
 {
+    public function q()
```

Comment:
What is the purpose of this?

Reply:
Oh yes, can be removed. I added it but after changed test method and it's no more used.

Reply:
> 
> 
> @mvorisek Please see test i added. It will throw PDO exception. Not sure what you mean by adding test.

I meant https://github.com/atk4/dsql/pull/179/files#diff-78123c449aa133c903d31fc8b34fb882R61 , but it needs to be tested against MySQL/Maria for that specific error code.